tests/Unit/ResponseTaggerTest.php

```diff
@@ -132,4 +132,15 @@ public function testNonStrictEmptyTag()
         $this->assertTrue($tagHandler->hasTags());
         $this->assertEquals('post-1', $tagHandler->getTagsHeaderValue());
     }
+
+    public function testUniqueTags()
+    {
+        $headerFormatter = new CommaSeparatedTagHeaderFormatter('FOS-Tags');
+
+        $tagHandler = new ResponseTagger(['header_formatter' => $headerFormatter, 'strict' => true]);
+        $tagHandler->addTags(['post-1', 'post-2', 'post-1']);
```

Comment:
for the sake of completeness, i'd like to have another line here that does `$tagHandler->addTags(['post-2', 'post-3']);`. with the current test, `array_merge($this->tags, array_unique($filtered))` would pass but its not correct.